package.json

```diff
@@ -65,7 +65,7 @@
   },
   "dependencies": {
     "@headlessui/react": "^1.5.0",
-    "@tippyjs/react": "4.2.5",
```

Comment:
@jinlee93 do you remember why you locked to a specific version when moving the Tooltip over?